COMET.Web.Common/ViewModels/Components/ParameterEditors/OrientationViewModel.cs

```diff
@@ -138,7 +138,7 @@ public Task OnMatrixValuesChanged(int index, string value)
         {
             var orientationMatrix = this.Orientation.Matrix;
 
-            if (double.TryParse(value, out var valueParsed))
+            if (double.TryParse(value, CultureInfo.InvariantCulture, out var valueParsed))
```

Comment:
This is unrelated to the original intent of the pull request but I included it here anyway.

I kept getting issues where this test wouldn't run VerifyOnMatrixValuesChanged because we were sending doubles with the format "1.0", which should take into consideration the current CultureInfo in order to be properly parsed. Setting this as a culture independent parsing will allow this to work, regardless if it's a comma or a dot.

cc. @samatrhea